File: Java/Selectionsort/Selectionsort.java
import java.util.Arrays;

public class Selectionsort {
    public static void main(String[] args) throws Exception {
        int[] numArray = {5,4,7,8,2,1,3,6};
        System.out.println("Unsorted Array: " + Arrays.toString(numArray));
        selectionSort(numArray);
        System.out.println("Result: " + Arrays.toString(numArray));
    }
    
    public static void selectionSort(int[] array) {
        for (int i = 0; i < array.length-1; i++) {
            int minElement = i;
            for (int j = i+1; j < array.length; j++) {
                if (array[j] < array[minElement]) {
                    minElement = j;
                }    
            }
            int temp = array[minElement];
            array[minElement] = array[i];
            array[i] = temp;
        }
    }


}

File: Python/Selectionsort/Selectionsort.py
def selectionSort(numArray):
    for i in range(0, len(numArray)):
        minElement = i
        for j in range(i+1, len(numArray)):
            if numArray[j] < numArray[minElement]:
                minElement = j
        temp = numArray[minElement]
        numArray[minElement] = numArray[i]
        numArray[i] = temp

if __name__ == "__main__":
    numArray=[5,4,7,8,2,1,3,6]
    selectionSort(numArray)
    for i in range(len(numArray)):
        print("%d" %numArray[i], end=" ")